src/constants.ts

```diff
@@ -137,6 +73,19 @@ export const ETH_TOKEN_LOGO_URI =
 
 export const POLYGON_SUPPORT_URL = "https://support.polygon.technology";
 
+export const POLYGON_TERMS_AND_CONDITIONS_URL = "https://polygon.technology/terms-of-use";
+
+export const POLYGON_PRIVACY_POLICY_URL = "https://polygon.technology/privacy-policy";
+
+export const POLYGON_ZKEVM_RISK_DISCLOSURES_URL =
+  "https://wiki.polygon.technology/docs/zkEVM/#polygon-zkevm-risk-disclosures";
+
+export const TOKEN_BLACKLIST = [
+  // WETH
```

Comment:
Blacklisting `WETH` is problematic because it's used by the price oracle in `src/contexts/price-oracle.context.tsx:91` and in `src/views/bridge-details/bridge-details.view.tsx:209` to get the price of ETH.

Reply:
Yes, we are not going to enable the fiat exchange rates for mainnet by the moment, this is addressed in #270.